pkg/recorder/upload_prod.go

```diff
@@ -1,19 +1,21 @@
+//go:build !test
 // +build !test
 
 package recorder
 
 import (
 	"bytes"
+	"cloud.google.com/go/storage"
 	"context"
 	"fmt"
-	"net/url"
-	"os"
-
 	"github.com/Azure/azure-storage-blob-go/azblob"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3"
+	"io"
+	"net/url"
+	"os"
```

Comment:
nit: can you run `goimports` (or just swap these with `cloud.google.com/go/storage` above